### PR TITLE
Feature/booking apt name

### LINKED ISSUE #52: Apt nr to be added in booking form.
As we need the apartment number in order to accept bookings and many of our users forget to write it, we would like to have it added to the requirements in the booking form.

### PR DESCRIPTION
**Pull request by klaseskilson**
This adds name and apartment numbers to bookings. Also, add a booking archive view and some minor booking related tweaks. This should close #52.

Pro tip: when merging, add `closes #52` to the merge commit message, and #52 will be closed.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -11,75 +11,77 @@
...
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
-  create_table "admins", force: true do |t|
-    t.string   "email",                  default: "", null: false
-    t.string   "encrypted_password",     default: "", null: false
-    t.string   "reset_password_token"
+  create_table "admins", force: :cascade do |t|
+    t.string   "email",                  limit: 255, default: "", null: false
```
Comment (klaseskilson) [inline db/schema.rb]: These `force` and `limit` changes were probably performed by Rake.

**Comment by klaseskilson** [comment]
Before merging this it is probably a good idea to add some unit tests to the booking model and controller, since it has grown in complexity.

**Comment by klaseskilson** [comment]
I'd say this is ready for review and merge. @CarlEnglund & @danielronnkvist, hit me!

**Comment by CarlEnglund** [comment]
In commit commit fe920b0 It says "Bokningen emottagen! Vi återkommer.". Some research on the word emottaga seems to indicate that this word is becoming deprecated over mottaga. " jfr MOTTAGA Emottaga, särskilt de finita formerna av detta verb, är numera nästan helt o. hållet inskränkt till skriftspr. o. är även där mindre vanl. än förr. I stället användas TAGA EMOT o. MOTTAGA." (http://g3.spraakdata.gu.se/saob/show.phtml?filenr=1/62/87.html). 

This is extremely minor and something you should not even care about, just had to find something in your otherwise awesome pull request. Should be able to merge it.

**Comment by klaseskilson** [comment]
@CarlEnglund, fixed!

**Comment by klaseskilson** [comment]
Sooo... can this be merged?

**Comment by klaseskilson** [comment]
chirp

@CarlEnglund!

**Comment by CarlEnglund** [comment]
It sure is!

**Pull request merged**